examples/basic_example.py

```diff
@@ -5,17 +5,24 @@
 def main():
     scenario = Scenario()
 
-    net = Network("10.0.0.0", "255.255.255.0")
+    net = Network("10.0.0.0", "255.255.255.0", base="0.0.0.2", delay="200ms")
+    net.block_ip_address("10.0.0.1")
 
     node1 = DockerNode('ping', docker_build_dir='./docker/ping')
     node2 = DockerNode('pong', docker_build_dir='./docker/pong')
-    net.connect(node1, node2, delay='200ms')
+    net.connect(node1, "10.0.0.17")
+    net.connect(node2)
 
     scenario.add_network(net)
 
+    @scenario.workflow
+    def increase_delay(workflow):
```

Comment:
I think these changes make the 'basic example' too advanced. Can you create a copy?